cmd/commands/cli/command.go

```diff
@@ -159,20 +168,112 @@ func (c *cliApp) handleActions(line string) {
 	}
 }
 
+func (c *cliApp) service(argsString string) {
+	printHelp := func() {
+		fmt.Println("service <action> [args]")
```

Comment:
we could just use a multiline string as a const, something like

```golang
const serviceHelp = `service <action> [args]
	start	<ProviderID> <ServiceType> [options]
...
`
```

The print then becomes way shorter, and does not clutter the method.

Reply:
Changed it.